### DISCUSSION TITLE
No platform authenticator available on this device

### DISCUSSION THREAD

**Discussion by shamelesshacker** [passlock-dev/svelte-passkeys#20] [Q&A]
Good day.

I was trying to register on Chrome / Debian Linux desktop and received the following message:

"No platform authenticator available on this device"

I was, however, able to register from my Android.

When I attempt to login from Debian desktop after registering,  I receive the same message, "No platform authenticator available on this device".

How can I login on Chrome desktop?

**Comment by thobson** [comment]
To be honest, I don't know enough about Chrome on Debian, but I think you may have run into [this issue.](https://developers.google.com/identity/passkeys/supported-environments?hl=en#linux) I'll look into why you can't use the Android passkey from desktop Chrome...
